contrib/kubespray/example/config.yml

```diff
@@ -67,4 +67,9 @@ docker_image_tag: <% latest-release %>
 
 # openpai_kubespray_extra_var:
 #   kay: value
-#   key: value
\ No newline at end of file
+#   key: value
+
+#######################################################################
+#                     host daemon port setting                        #
+#######################################################################
+# host_daemon_port_range: "40000 60000"
```

Comment:
Although it works, I suggest to seprate the value into 2 parts and both of the have a default value. Because setting range in 1 vars is rarely seen

Reply:
fixed

Reply:
> Although it works, I suggest to seprate the value into 2 parts and both of the have a default value. Because setting range in 1 vars is rarely seen

this is exactly what systemd does, it's one config entry

Reply:
Anyway, all has same effect. Separate  config is more clear for me